cis_v120/docs/cis_v120_1_11.md

```diff
@@ -0,0 +1,30 @@
+## Description
+
+IAM password policies can require passwords to be rotated or expired after a given number of days. It is recommended that the password policy expire passwords after 90 days or less.
+
+Reducing the password lifetime increases account resiliency against brute force login attempts. Additionally, requiring regular password changes help in the following scenarios:
+
+  - Passwords can be stolen or compromised sometimes without your knowledge. This can happen via a system compromise, software vulnerability, or internal threat.
+  - Certain corporate and government web filters or proxy servers have the ability to intercept and record traffic even if it's encrypted.
+  - Many people use the same password for many systems such as work, email, and personal.
+  - Compromised end user workstations might have a keystroke logger.
```

Comment:
```suggestion

- Passwords can be stolen or compromised sometimes without your knowledge. This can happen via a system compromise, software vulnerability, or internal threat.
- Certain corporate and government web filters or proxy servers have the ability to intercept and record traffic even if it's encrypted.
- Many people use the same password for many systems such as work, email, and personal.
- Compromised end user workstations might have a keystroke logger.
```